R/zzz.R

```diff
@@ -23,12 +23,12 @@
 #'
 #' @inheritParams check_package_version
 #'
-#' @eval typed_param(callback, function(...) ANY,
-#'   "A callback to execute upon package load. Note that no arguments are
+#' @param callback (`function(...) ANY`)\cr
+#'   A callback to execute upon package load. Note that no arguments are
 #'   passed to this function. Any necessary data must be provided upon
-#'   construction.")
+#'   construction.
 #'
-#' @eval typed_param(message, NULL | character(1L),
+#' @param message (`NULL` or `character[1]`)\cr
```

Comment:
```suggestion
#' @param message (`NULL` or `string`)\cr
```
this is our current convention :-)